merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -154,15 +168,17 @@ public List<String> validateModelArguments(final String missionModelId, final Ma
          MissionModelLoadException,
          InvalidArgumentsException
   {
-    return this.loadConfiguredMissionModel(missionModelId)
-               .validateConfiguration(arguments);
+    return this.loadUnconfiguredMissionModel(missionModelId)
+               .validateMissionModelArguments(arguments);
```

Comment:
`MissionModelFacade#validateConfiguration` is no longer used anywhere. Can we remove it?

Reply:
Removed